examples/demo.rs

```diff
@@ -20,20 +18,16 @@ use std::path::Path;
 extern crate petgraph;
 use self::petgraph::{Graph};
 
-//fn resized(w:u32,h:u32) {width=w; height=h;}
-
-#[derive(Debug)]
-struct NodeState {
+#[derive(Debug, Clone)]
+struct DemoNode {
+    name: String,
     position: [f64; 2]
 }
 
-#[derive(Debug)]
-struct Edge;
-impl EditableEdge for Edge {
-    fn default() -> Self { Edge }
-}
-
-impl EditableNode for NodeState {
+impl EditableNode for DemoNode {
+    fn get_label(&self) -> &str {
```

Comment:
We could also add default impls for a lot of the trait methods, if some don't make sense in every situation.